packages/manager/src/views/WebPropertyEnvPage/WebPropertyEnvPage.tsx

```diff
@@ -785,7 +788,7 @@ export const WebPropertyEnvPage = (): JSX.Element => {
         <SyncServiceForm
           propertyIdentifier={propertyIdentifier}
           onClose={() => handlePopUpClose('updateSync')}
-          env={envList?.data?.length ? envList.data : []}
+          env={envWithEphList || []}
```

Comment:
env={envWithEphList} 
we can keep it like this because we're already sending the [] from envWithEphList